.gitignore

```
@@ -8,3 +8,4 @@ output/*
 .identity
 .netrc
 
+**pulled-images.txt
```

```suggestion
**/pulled-images.txt
```
Does this need to have a slash in it?

---

I think this also works... my `git status` stopped mentioning the file with it as it was. That said, it also doesn't complain with the slash, so I pushed up a commit with this change.